src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/PayRatesService.kt

```diff
@@ -175,6 +181,7 @@ class PayRatesService(
             .toCourseActivityPayRate(
               courseActivity = existingActivity,
               startDate = existingActivity.scheduleStartDate,
+              endDate = it.endDate,
```

Comment:
When creating the new pay rate with an earlier start date bear in mind it is not necessarily active - it could have an end date.